lib/Sabberworm/CSS/Parser.php

```diff
@@ -56,7 +59,7 @@ public function __construct($sText, Settings $oParserSettings = null) {
 
 	public function setCharset($sCharset) {
 		$this->sCharset = $sCharset;
-		$this->iLength = $this->strlen($this->sText);
+		$this->iLength = count($this->aText);
```

Comment:
If it so, why is `bMultibyteSupport` flag needed? Can I pass `bMultibyteSupport = false` and `@charset "UTF-8"`?